EightBall.java

```diff
@@ -0,0 +1,17 @@
+package com.fortify.sca.plugins.maven.samples;
+
+import java.io.FileReader;
+
+public class EightBall {
+	public static void main(String args[]) throws Exception {
+		char[] buffer = new char[1024];
+		String filename = args[0];
+		try {
+			filename = "" + (Integer.parseInt(filename) % 3);
+		} catch (Exception e) {
+			System.out.println("Invalid input.");
```

Comment:
<p><b>Security Scanning</b> / Fortify SAST</p>
<h3>Low - Unchecked Return Value </h3>
<p>The method main() in EightBall.java ignores the value returned by read() on line 12, which could cause the program to overlook unexpected states and conditions.


</p>
<br>
<p><a href=http://localhost:2320/ssc/html/ssc/version/46/fix/7250/?engineType=SCA&issue=F7A138CDE5235351F6A4405BA4AD7C53&filterSet=f2f88863-0d8f-4d22-b001-3e1cd3ec0f45 target="_blank" rel="noopener noreferrer">More detailed information</a></p>